homeassistant/helpers/config_validation.py

```diff
@@ -591,7 +591,7 @@ def validator(value):
     vol.Optional(CONF_ALIAS): string,
     vol.Required("delay"): vol.Any(
         vol.All(time_period, positive_timedelta),
-        template)
+        template, {match_all: template_complex})
```

Comment:
I don't think `match_all` is correct

Reply:
Hmm, seems to work equally either way, will remove it.